guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -39,8 +39,12 @@ endif::[]
 ifdef::satellite[]
 [NOTE]
 ====
-{Project} supports DEFAULT and FIPS crypto-policies.
+* {Project} supports DEFAULT and FIPS crypto-policies.
 The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+* In-place upgrade of {EL} systems in FIPS mode is not supported by {Team}.
+Turning FIPS off, upgrading from {EL} 7 to {EL} 8, and then turning FIPS on is not supported either.
+Instead, migrate the {Project} {ProductVersion} to a freshly installed {EL} 8 system with FIPS mode enabled.
```

Comment:
```suggestion
Instead, migrate {Project} {ProductVersion} to a freshly installed {EL} 8 system with FIPS mode enabled.
```

I'd drop the `the`, but it might be me?

Reply:
Yeah, I am a little conflicted about it too. I felt like adding the article because we are now referring to the existing installation. @maximiliankolb, what do you think?

Reply:
We generally don't use articles. Maybe reword to "migrate your "{ProjectServer}"? "Your" is often our alternative to "the" :D

Reply:
@evgeni @maximiliankolb done! :D